tests/ui/parser/deli-ident-issue-2.stderr

```diff
@@ -4,16 +4,10 @@ error: mismatched closing delimiter: `]`
 LL |     if 1 < 2 {
    |              ^ unclosed delimiter
 LL |         let _a = vec!];
-   |                      ^ mismatched closing delimiter
+   |                      ^
+   |                      |
+   |                      mismatched closing delimiter
```

Comment:
I think we can remove this `mismatched closing delimiter` if we reported `missing open `[` for this delimiter` on this span here.